requirements/quality.txt

```diff
@@ -224,7 +224,7 @@ requests==2.28.1
     #   twine
 requests-toolbelt==0.10.0
     # via twine
-responses==0.21.0
+responses==0.22.0
```

Comment:
No breaking change in the new version https://github.com/getsentry/responses/blob/master/CHANGES